test/registry.test.js

```diff
@@ -63,6 +63,11 @@ describe('AtlasRegistry', () => {
     assert.isFalse(r.started);
   });
 
+  it('start with no stop should not prevent the process from exiting', () => {
+    const r = new AtlasRegistry({uri: 'http://localhost/foo'});
+    r.start();
```

Comment:
Do each of these tests run in a separate node process? Or is this the only test that starts without stopping and so the build would just timeout?

Does node have something like the Java shutdown hook or python `atexit` that could be used to trigger shutdown automatically when the VM is stopped?